tasks/pip.yml

```diff
@@ -130,36 +114,23 @@
   tags:
     - molecule-idempotence-notest
 
-- name: Install django-cors-headers
-  when: openwisp2_django_cors.get('enabled')
-  pip:
-    name: "django-cors-headers~=4.4.0"
-    state: latest
-    virtualenv: "{{ virtualenv_path }}"
-    virtualenv_python: "{{ openwisp2_python }}"
-  retries: 5
-  delay: 10
-  register: result
-  until: result is success
-  notify: Reload application
-
 - name: Install extra python packages
   pip:
     name: "{{ openwisp2_extra_python_packages }}"
     state: latest
     virtualenv: "{{ virtualenv_path }}"
     virtualenv_python: "{{ openwisp2_python }}"
-  retries: 5
-  delay: 10
+  when: openwisp2_extra_python_packages | length > 0
+  retries: 3
+  delay: 5
   register: result
   until: result is success
   notify: Reload application
   tags: [extra_pip]
 
-- name: Install static minification dependencies
+- name: Install django
   pip:
-    name:
-      - django-pipeline~=3.1.0
+    name: "{{ openwisp2_django_version }}"
```

Comment:
Not sure but can we move Django also in `requirements.txt` file?
cc @nemesifier

Reply:
Not for now, because that's configurable.